repo/meta/schema/package-schema.json

```diff
@@ -64,6 +64,23 @@
         "type": "string",
         "pattern": "^[^\\s]+$"
       }
+    },
+    "licenses": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
```

Comment:
This type should have the `"required": ["name", "url"]` property as well, otherwise an array of empty objects would be valid.